Client/src/main/java/edu/sdccd/cisc191/template/Graphics.java

```diff
@@ -271,9 +266,7 @@ public void player2Wins() {
             System.out.println("Player 2 wins!\n" + response.getData());
             saveScores(); // Save scores after updating
         });
-        task.setOnFailed(e -> {
-            System.out.println("Error updating Player 2 score");
-        });
+        task.setOnFailed(e -> System.out.println("Error updating Player 2 score"));
```

Comment:
Changed the statement lambda to an expression lambda. If there was more than one statement you would need to use a statement lambda. In this case there is only one so an expression lambda will work and keep it to one line of code.